docker/pravega/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+
+#FROM openjdk:8u212-jre-alpine3.9
+ARG DOCKER_IMAGE_REGISTRY
+ARG DOCKER_IMAGE_REPOSITORY
+ARG DOCKER_IMAGE_VERSION
+ARG PRAVEGA_IMAGE="${DOCKER_IMAGE_REGISTRY}/${DOCKER_IMAGE_REPOSITORY}:${DOCKER_IMAGE_VERSION}"
+FROM ${PRAVEGA_IMAGE}
+WORKDIR /opt/pravega
+ARG PRAVEGA_BINDINGS_VERSION
+ARG HDFS_JAR="hdfs/build/libs/pravega-hdfs-${PRAVEGA_BINDINGS_VERSION}.jar"
+ARG S3_JAR="extendeds3/build/libs/pravega-extendeds3-${PRAVEGA_BINDINGS_VERSION}.jar"
```

Comment:
Where are we removing old bindings jar ?

Reply:
Yes Sachin, it was there. Let me check. Thanks,

Reply:
https://github.com/pravega/pravega-storage-adapters/pull/15/commits/cf93b19805b85a60d7889e4325125cb12e84e2ef#diff-ec695cab72c3c30c1f15b1bcd0dafbfcR23

Reply:
Line 24